crates/oxide/src/lib.rs

```diff
@@ -24,7 +24,7 @@ pub mod paths;
 pub mod scanner;
 
 static SHOULD_TRACE: sync::LazyLock<bool> = sync::LazyLock::new(
-    || matches!(std::env::var("DEBUG"), Ok(value) if value.eq("*") || value.eq("1") || value.eq("true") || value.contains("tailwind")),
+    || matches!(std::env::var("DEBUG"), Ok(value) if value.eq("*") || (value.contains("tailwindcss:oxide") && !value.contains("-tailwindcss:oxide"))),
```

Comment:
When using `DEBUG=1` it also shows a lot of oxide related tracing logs. This change only shows them if you are using `DEBUG=*` or `DEBUG=tailwindcss:oxide` by following the `DEBUG` conventions.